### PR TITLE
Migrate to Bingo for tool dependencies

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

File: test/integration.sh [hunk 1]
```diff
@@ -9,7 +9,16 @@ set -euo pipefail
...
 
-(./tmp/bin/dex serve ./test/config/dex.yaml) &
+OBSERVATORIUM=${OBSERVATORIUM:=./observatorium}
+DEX=${DEX:-dex}
+PROMETHEUS=${PROMETHEUS:-prometheus}
+THANOS=${THANOS:-thanos}
+LOKI=${LOKI:-loki}
+OPA=${OPA:-opa}
+UP=${UP:-up}
+WEBSOCAT=${WEBSOCAT:=websocat}
```
Comment (periklis) [inline test/integration.sh:19]: You can eliminate passing this variables here by sources `variables.env` here.

File: test/load.sh [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 set -euo pipefail
 
-BIN_DIR=./tmp/bin
 DATA_DIR=./docs/loadtests/results
 DOC_DIR=./docs
 
+STYX=${STYX:-styx}
+PROMETHEUS=${PROMETHEUS:-prometheus}
+MOCKPROVIDER=${MOCKPROVIDER:-mockprovider}
+PROMREMOTEBENCH=${PROMREMOTEBENCH:-promremotebench}
```
Comment (periklis) [inline test/load.sh:10]: Same here, you can replace these by including `variables.env`.

File: test/run-local.sh [hunk 3]
```diff
@@ -8,7 +8,12 @@ set -euo pipefail
 result=1
 trap 'kill $(jobs -p); exit $result' EXIT
 
-(./tmp/bin/dex serve ./test/config/dex.yaml) &
+OBSERVATORIUM=${OBSERVATORIUM:=./observatorium}
+DEX=${DEX:-dex}
+THANOS=${THANOS:-thanos}
+LOKI=${LOKI:-loki}
```
Comment (periklis) [inline test/run-local.sh:14]: Same here, you can replace these by sourcing `variables.env`

File: test/run-local.sh [hunk 4]
```diff
@@ -20,7 +25,7 @@ until curl --output /dev/null --silent --fail --insecure https://127.0.0.1:5556/
 done
 
 (
-  ./observatorium \
+  ${OBSERVATORIUM} \
```
Comment (periklis) [inline test/run-local.sh:28]: nit. you don't need curly braces here if you don't use any of its fancy features, e.g. default if empty, etc.

**Review by periklis — APPROVED** [review]
Looks overall good time. Marking only minor improvements to the author to include or not for now.

File: test/integration.sh (hunk 1 above)
Comment (kakkoyun) [reply test/integration.sh:19]: That would be super awesome. However we have an issue concerning that https://github.com/bwplotka/bingo/issues/40. I haven't found enough motivation to fix it yet. cc @bwplotka

**Review by periklis — COMMENTED** [review]
Awesome!

**Pull request merged**